src/components/ui/PrismCode.tsx

```diff
@@ -1,27 +1,43 @@
 import React from 'react';
 import Highlight, { defaultProps, Language, PrismTheme } from 'prism-react-renderer';
-import { Text, TextVariants } from '@patternfly/react-core';
+import { Text, TextVariants, ClipboardCopy, clipboardCopyFunc } from '@patternfly/react-core';
 import defaultTheme from 'prism-react-renderer/themes/github';
 import './PrismCode.css';
-
 type PrismCodeProps = {
   code: string;
   language?: Language;
   theme?: PrismTheme;
+  copiable?: boolean;
 };
-
-const PrismCode: React.FC<PrismCodeProps> = ({ code, language = 'bash', theme = defaultTheme }) => (
+const PrismCode: React.FC<PrismCodeProps> = ({
+  code,
+  language = 'bash',
+  theme = defaultTheme,
+  copiable = false,
+}) => (
   <Highlight {...defaultProps} code={code} language={language} theme={theme}>
     {({ className, style, tokens, getLineProps, getTokenProps }) => (
-      <Text component={TextVariants.pre} className={className} style={style}>
-        {tokens.map((line, i) => (
-          <div key={i} {...getLineProps({ line, key: i })}>
-            {line.map((token, key) => (
-              <span key={key} {...getTokenProps({ token, key })} />
-            ))}
-          </div>
-        ))}
-      </Text>
+      <>
+        <Text component={TextVariants.pre} className={className} style={style}>
+          {copiable && (
+            <ClipboardCopy
+              isReadOnly
+              variant={'inline-compact'}
+              onCopy={(e) => clipboardCopyFunc(e, code)}
+              style={{ float: 'right', background: 'inherit' }}
+            >
+              {}
+            </ClipboardCopy>
```

Comment:
@jgyselov does this work?

```suggestion
            <ClipboardCopy
              isReadOnly
              variant={'inline-compact'}
              onCopy={(e) => clipboardCopyFunc(e, code)}
              style={{ float: 'right', background: 'inherit' }}
              children={null}
            />
```

Reply:
No. I get "semantic error TS2741: Property 'children' is missing in type ... but required in type ...".

Reply:
Bummer... How about this?

```suggestion
            <ClipboardCopy
              isReadOnly
              variant={'inline-compact'}
              onCopy={(e) => clipboardCopyFunc(e, code)}
              style={{ float: 'right', background: 'inherit' }}
              children={null}
            />
```

Reply:
@jkilzi That does compile, but the linter doesn't look happy.